README.md

```diff
@@ -36,4 +40,29 @@ Done!! You should see the following page if everything is successfull.
 
 ![SuccessfullyInstalledApp](https://raw.githubusercontent.com/thoth-station/Qeb-Hwt/master/docs/images/SuccessfullyInstalledApp.png)
 
-Long Live Bots!
\ No newline at end of file
+Long Live Bots!
+
+
+Usage
+======
+
+Qeb-Hwt is one of the integration of Thoth (LINK to other integrations). 
```

Comment:
LINK :)

and maybe we should explain what the runtime_env means and how it influences results/advises

Reply:
and also what is expected by Thoth when there is pipenv or pip as `requirements format`